src/libraries/System.Private.CoreLib/src/ILLink/ILLink.Substitutions.Shared.xml

```diff
@@ -10,6 +10,7 @@
          to be trimmed when Invariant=true, and allows for the Settings static cctor (on Unix) to be preserved when Invariant=false. -->
     <type fullname="System.Globalization.GlobalizationMode">
       <method signature="System.Boolean get_Invariant()" body="stub" value="true" feature="System.Globalization.Invariant" featurevalue="true" />
+      <method signature="System.Boolean get_PredefinedCulturesOnly()" body="stub" value="true" feature="System.Globalization.PredefinedCulturesOnly" featurevalue="true" />
```

Comment:
@eerhardt could you please have a look at the change in this file and the change in the test .proj file too and confirm it is correct? thanks!

Reply:
Hmm, I think this is a bit problematic as it will need SDK update to pass the feature (System.Globalization.PredefinedCulturesOnly) value based on other feature (System.Globalization.Invariant) value

Reply:
@marek-safar Marek Safar FTE @eerhardt Eric Erhardt FTE could you please advise if I need to do anything here? should I hold this change? or SDK changes can be done after I merge this change? anything else in general I need to do?

Reply:
> or SDK changes can be done after I merge this change?

Yes, once we merge this change into runtime, we can make the change to the SDK

Reply:
I have created the SDK draft PR https://github.com/dotnet/sdk/pull/18672. and added the entry to the document https://github.com/dotnet/runtime/blob/main/docs/workflow/trimming/feature-switches.md